clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -121,6 +121,13 @@ New checks
   Gives warnings for tagged unions, where the number of tags is
   different from the number of data members inside the union.
 
+- New :doc:`bugprone-nondeterministic-pointer-iteration-order
+  <clang-tidy/checks/bugprone/nondeterministic-pointer-iteration-order>`
+  check.
+
+  Detect certain nondeterministic pointer seen with unordered
```

Comment:
Please synchronize with first statement in documentation.

Reply:
I'll address, thank you